docs/README.md

```diff
@@ -110,10 +111,27 @@ functions:
     env:
       MY_VARIABLE: "my-value"
 ```
+### Managing containers
+
+Your containers must be defined in the `custom` field of your manifest. You need to specify the relative path of your container directory containing the Dockerfile and any additional files in `directory` :
+
+
+```yml
+custom:
+  myContainer:
+    directory: my-container-directory
+    # Environment only available in this function
+    env:
+      MY_VARIABLE: "my-value"
+```
+
+Your containers must EXPOSE PORT 8080 and listen on it.
```

Comment:
Add:
Scaleway's platform will automatically inject a `PORT` environment variable on which your server should be listening for incoming traffic. By default, this PORT is 8080